ArduPlane/Parameters.h

```diff
@@ -174,7 +174,7 @@ class Parameters {
         //
         k_param_airspeed_min = 120,
         k_param_airspeed_max,
-        k_param_FBWB_min_altitude_cm,  // 0=disabled, minimum value for altitude in cm (for first time try 30 meters = 3000 cm)
+        k_param_FBWB_min_altitude,  // 0=disabled, minimum value for altitude in cm (for first time try 30 meters = 3000 cm)
```

Comment:
The param is now "CRUISE" - so should be variable be k_param_cruise_min_alt? (note left to right naming ;-) )
Also the comment says it's in "cm" which it is not any longer.

Reply:
fixed